cmd/root.go

```diff
@@ -125,6 +129,29 @@ func AddCommands(cmd *cobra.Command) {
 	cmd.PersistentFlags().IntVar(&config.HARE.ExpectedLeaders, "hare-exp-leaders",
 		config.HARE.ExpectedLeaders, "The expected number of leaders in the hare protocol")
 
+	/**======================== PoST Flags ========================== **/
+
+	cmd.PersistentFlags().StringVar(&config.POST.DataDir, "post-datadir",
+		config.POST.DataDir, "The directory that contains post data files")
+	cmd.PersistentFlags().Uint64Var(&config.POST.SpacePerUnit, "post-space",
+		config.POST.SpacePerUnit, "Space per unit, in bytes")
```

Comment:
i don't understand the meaning of space unit, especially since we have space per file, can you elaborate or rephrase?

Reply:
Space per unit refers to the protocol. `filesize` is just for dividing it to multiple files, locally.
We’ll revisit the terminology after we’ll add support to multiple ids/units.